examples/grpc/grpc.arrai

```diff
@@ -1,21 +1,21 @@
 (
     type: //fn.fix(\type \t
-        t.type cond (
-            'primitive': t.primitive cond (
-                'DECIMAL':  'double',
-                'INT':      'int64',
-                'FLOAT':    'double',
-                'STRING':   'string',
-                'STRING_8': 'string',
-                'BOOL':     'bool',
-                'DATE':     'string',
-                'DATETIME': 'string',
-            ),
-            'sequence': 'repeated ' + type(t.sequence)
-            *: cond (
+        t.type cond {
+            ('primitive'): t.primitive cond {
+                ('DECIMAL'):  'double',
+                ('INT'):      'int64',
+                ('FLOAT'):    'double',
+                ('STRING'):   'string',
+                ('STRING_8'): 'string',
+                ('BOOL'):     'bool',
+                ('DATE'):     'string',
+                ('DATETIME'): 'string',
```

Comment:
I really don't want to force this on people. Let's add string and character literals alongside number literals as special cases not requiring parentheses. Expressions in general should still require them.